datahub-graphql-core/src/main/resources/ingestion.graphql

```diff
@@ -92,6 +92,11 @@ type ExecutionRequestSource {
   The type of the source, e.g. SCHEDULED_INGESTION_SOURCE
   """
   type: String
+
+  """
+  The urn of the ingestion source
+  """
+  ingestionSource: String
```

Comment:
Since this is an URN and not the actual resolved ingestionSource entity object, I'd recommend naming this `ingestionSourceUrn` for clarity and consistency